src/components/OtherButtons/OtherButtons.tsx

```diff
@@ -0,0 +1,30 @@
+import { faUpload} from '@fortawesome/free-solid-svg-icons';
+import { faDownload} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+
```

Comment:
1 rzecz, to używaj prettier i eslinta, bo widzę, że nie ma tutaj odpowiednich wcięć, odstępów itp. Prettier podczas zapisywania posortuje Ci cały kod wedle schematu narzuconego w konfiguracji, która jest zapisana w projekcie